src/runner.ts

```diff
@@ -140,5 +146,10 @@ export async function run(fn: Runner, args: string[], options: DetectOptions = {
     return
   }
 
-  await ezspawn(command, { stdio: 'inherit', cwd })
+  await x(command, undefined, {
```

Comment:
should we use arguments here (second argument in x call)?

Reply:
It'll default to an empty array under the hood if you pass undefined like this. So should be fine

Reply:
looks like `tinyexec` doesn't support command string mode, check https://github.com/antfu-collective/ni/issues/224

For example, check bot actions here https://github.com/unocss/unocss/actions/runs/10388099755/job/28762993666